pallets/asset-manager/src/lib.rs

```diff
@@ -396,6 +397,18 @@ pub mod pallet {
 		}
 	}
 
+	impl<T: Config> Contains<<T::AssetConfig as AssetConfig<T>>::AssetLocation> for Pallet<T> {
+		fn contains(location: &<T::AssetConfig as AssetConfig<T>>::AssetLocation) -> bool {
+			LocationAssetId::<T>::contains_key(&location)
+		}
+	}
+
+	impl<T: Config> GetByKey<<T::AssetConfig as AssetConfig<T>>::AssetLocation, u128> for Pallet<T> {
+		fn get(location: &<T::AssetConfig as AssetConfig<T>>::AssetLocation) -> u128 {
+			todo!();
```

Comment:
Can you log this todo as an issue please.

Reply:
I will implement it in this pr.